packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -213,6 +241,7 @@ describe("Garbage Collection Tests", () => {
 
 		const summarizerContainerTests = (
 			timeout: number,
+			mode: "inactive" | "sweep",
```

Comment:
Future - could remove the following fooEventName params and compute them from this mode.